api/capi/include/nnstreamer.h

```diff
@@ -885,6 +904,43 @@ int ml_pipeline_element_get_property_double (ml_pipeline_element_h elem_h, const
  */
 int ml_pipeline_element_get_property_enum (ml_pipeline_element_h elem_h, const char *property_name, uint32_t *value);
 
+/****************************************************
+ ** NNStreamer Pipeline tensor_if Control          **
+ ****************************************************/
+/**
+ * @brief Registers a tensor_if custom callback.
+ * @details If the if-condition is complex and cannot be expressed with tensor_if expressions, you can define custom condition.
+ * @since_tizen 6.5
+ * @remarks If the function succeeds, @a tensor_if handle must be released using ml_pipeline_if_custom_unregister().
+ * @param[in] name The name of tensor_if
```

Comment:
Is this ```name```, the name of tensor-if element? or the name of custom condition?

Reply:
Name of the tensor_if element.

Reply:
Why do  you need the name of tensor_if element?

From tensor_if.h:
```
 *
 * ...
 * // Register custom callback function describing the condition
 * nnstreamer_if_custom_register ("tifx", tensor_if_custom_cb, NULL);
 * ...
 * // Use the condition in a pipeline.
 * // E.g., Pipeline of " ... ! tensor_if compared-value=CUSTOM compared-value-option=tifx then=TENSORPICK then-option=0 else=TENSORPICK else-option=1 ! ... "
 * ...
 * // After everything is done.
 * nnstreamer_if_custom_unregister ("tifx");
 * @endcode
```

Reply:
Oh my.. You're right. Updated.